lldb/source/Core/Disassembler.cpp

```diff
@@ -658,8 +658,13 @@ void Instruction::Dump(lldb_private::Stream *s, uint32_t max_opcode_byte_size,
       // the byte dump to be able to always show 15 bytes (3 chars each) plus a
       // space
       if (max_opcode_byte_size > 0)
-        m_opcode.Dump(&ss, max_opcode_byte_size * 3 + 1);
-      else
+        // make RISC-V opcode dump look like llvm-objdump
```

Comment:
If you say instead "Pretty print RISC-V opcodes in the same format that llvm-objdump uses." this hints at the motivation and what keywords to look for in llvm-objdump's source.